course_discovery/apps/api/serializers.py

```diff
@@ -631,8 +631,9 @@ class CourseRunWithProgramsSerializer(CourseRunSerializer):
     programs = serializers.SerializerMethodField()
 
     @classmethod
-    def prefetch_queryset(cls, queryset=None):
+    def prefetch_queryset(cls, queryset=None, edly_sub_org=None):
```

Comment:
As discussed we will use `edly-org` and `edly-sub-org` for URL's and URL based restrictions and use edx organizations `short_name` (org key used in course run id) for filtering courses/course runs.

`edly_sub_org` -> `edx_org_short_name`

Reply:
Done